src/math/modint.hpp

```diff
@@ -46,4 +46,12 @@ struct Modint {
     mint operator*(const mint rhs) const { return mint(*this) *= rhs; }
     mint operator/(const mint rhs) const { return mint(*this) /= rhs; }
     mint operator-() const { return mint() - *this; }  // 単項
-};
\ No newline at end of file
+    // 入出力ストリーム
+    friend ostream& operator<<(ostream& os, const mint& p) { return os << p.v; }
+    friend istream& operator>>(istream& is, mint& p) {
+        int64_t t;
```

Comment:
```suggestion
    long long t;
```
これは私のミスなのですが、データ型を統一するために `long long` を使った方がいいですね、すみません。

Reply:
なるほど。確かにそうですね。